assets/src/components/inputModals/createPresetModal.tsx

```diff
@@ -24,8 +24,12 @@ const CreatePresetModal = ({
           event.preventDefault()
           const existingPreset = findPresetByName(routeTabs, presetName)
           if (existingPreset) {
+            window.FS?.event(
+              "User tried to Save new Preset under existing Name"
```

Comment:
question (non-blocking): Is this necessary if the save/cancel overwrite events are separately tracked? If so, the wording "User tried" makes me think they tried but didn't succeed. Maybe something like "User trying to..." since there are still confirmation steps to take?

Reply:
I don't think it's nescisary, and it wasn't in AC, so I'm open to changing it. I had a feeling that this might be useful as a metric which is why I added it (and it was easy with how tests were configured).

Reply:
Maybe this?
> User manually typed existing Saved Preset name

Trying to think about what would be most descriptive and helpful when looking at this on the fullstory timeline or search has been helpful but I'm still unsure what is the best descriptor for what is happening.

Reply:
Heh I've come up with a few

> User manually typed existing Saved Preset name
> User trying to save over existing Saved Preset
> User submitted name of existing preset when creating a new Saved Preset

Reply:
PTALA 🙏🏻 6e5975e9f57bddb9da40e8afc4a811e27f067dc3

Reply:
LGTM!